configs/sites/tier1/aws-pcluster/README.md

```diff
@@ -223,7 +222,7 @@ exit
 mkdir -p /home/ubuntu/jedi/ecflow-5.8.4/src
 cd /home/ubuntu/jedi/ecflow-5.8.4/src
 wget -O ecFlow-5.8.4-Source.tar.gz \
-     https://confluence.ecmwf.int/download/attachments/8650755/ecFlow-5.8.4-Source.tar.gz?api=v2
+     https://github.com/ecmwf/ecflow/releases/download/-5.8.4/ecFlow-5.8.4-Source.tar.gz
```

Comment:
Is this link correct? I get a "not found"

Reply:
Good point... https://github.com/ecmwf/ecflow/archive/refs/tags/5.8.4.tar.gz maybe or keep the previous one?

Reply:
Good catch. I've updated it. If you're interested in what is going on here, read below.

I was messing with this section because I was trying out the recent ecflow 5.12 which is downloaded using GitHub's release feature. I tried the newer version to find a fix for Fabio, but the error he was experiencing turned out to be an environment issue unique to his computer, so I aborted the update attempt since without further testing and verification I didn't want to update ecflow. That said, GitHub's pipes are way bigger, making this a much faster download location. The proper link for 5.8.4 is associated with GitHub's tag based packaging instead of their release packaging.